com.empesol.timetracker.yaml

```diff
@@ -0,0 +1,56 @@
+id: com.empesol.timetracker
+app-id: com.empesol.timetracker
+runtime: org.freedesktop.Platform
+runtime-version: 24.08
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+command: app.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --env=INSTALL_TYPE=flatpak
```

Comment:
where does that come from and why is needed?

Reply:
network - data transfer via web service (workday and tasks data)
x11 - java currently support only x11
ipc - added because the lint processor required this in combination with x11
dri - java app is using gpu for drawing
INSTALL_TYPE=flatpak - environment variable to set that it's installed via flatpak

Reply:
>  INSTALL_TYPE=flatpak - environment variable to set that it's installed via flatpak

And where is this used?

Reply:
Currently is not in use. It is intended for use within the application, for example, to show a message in app that it should be updated via Flatpak.

Reply:
Env INSTALL_TYPE removed, there are other ways to detect flatpak.